tempesta_fw/http.c

```diff
@@ -498,6 +498,45 @@ tfw_http_set_hdr_connection(TfwHttpMsg *hm, int conn_flg)
 	}
 }
 
+/*
+ * Add/Replace/Remove Keep-Alive header field to/from HTTP message.
+ */
+static int
+tfw_http_set_hdr_keep_alive(TfwHttpMsg *hm, int conn_flg)
+{
+	int ret;
+
+	if ((hm->flags & __TFW_HTTP_CONN_MASK) == conn_flg)
+		return 0;
+
+	switch (conn_flg) {
+	case TFW_HTTP_CONN_CLOSE:
+		ret = TFW_HTTP_MSG_HDR_DEL(hm, "Keep-Alive", TFW_HTTP_HDR_RAW);
+		return (ret == -ENOENT) ? 0 : ret;
+	case TFW_HTTP_CONN_KA:
+		/*
+		 * If present, "Keep-Alive" header informs the other side
+		 * of the timeout policy for a connection. Otherwise, it's
+		 * presumed that default policy is in action.
+		 *
+		 * TODO: Add/Replace "Keep-Alive" header when Tempesta
+		 * implements connection timeout policies and the policy
+		 * for the connection differs from default policy.
+		 */
```

Comment:
Well, for instance Nginx's ```keepalive_timeout``` option is very useful for DDoS protection, so it's good to introduce the option now for Tempesta. Also don't forget to updatae README.